.travis/matrix-install.sh

```diff
@@ -5,27 +5,23 @@ source .travis/utils.sh
 
 echo "Matrix params: MATRIX_TYPE=${MATRIX_TYPE:?Empty value for MATRIX_TYPE}, BOWER=${BOWER:-no}"
 
-if [ "${MATRIX_TYPE}" = "python" ]; then
-    pip install --exists-action w --timeout 60 --requirement=requirements/test-requirements.txt
+if [ "${MATRIX_TYPE}" = "python" ] || [ "${MATRIX_TYPE}" = "python-sharded" ]; then
 
-    setup_elasticsearch
+    sleep 10  # kafka is slow to start up
```

Comment:
Possibly. It would have to be a more complex check since it does accept connections but if you try to create a topic you get an error. Can certainly look into a better option but just wanted to get this in now rather than delay further.